doc/manuals/telesculptor.rst

```diff
@@ -1,12 +1,58 @@
+.. _telesculptor:
```

Comment:
This file is a special case since it is currently what is built by CMake into a static webpage that is installed along side TeleSculptor.  That build currently uses `docutils` but not full blown sphinx processing so I don't think it will handle this new syntax correctly.  This file also has some content overlap with the rest of the manual.  I suggest we take this file out of the TOC and revert these changes for the initial pass.  This way the file will continue to work with the `docutils` build.  Once we have this initial documentation on readthedocs I would like to do the following:

1. Update the TeleSculptor build to use sphinx and build/install this new manual instead.
2.  Refactor the content from this file to better fit into the new manual.

These would both be after this current branch lands

Reply:
Note: you may want to cherry-pick the changes you made to this file onto a new branch.  Eventually we will need them, we just aren't ready for them yet.